lang/derive/accounts/src/lib.rs

```diff
@@ -418,6 +419,44 @@ use syn::parse_macro_input;
 ///                 </code></pre>
 ///             </td>
 ///         </tr>
+///         <tr>
+///             <td>
+///                 <code>#[account(realloc = &lt;space&gt;, realloc::payer = &lt;target&gt;, realloc::zero = &lt;bool&gt;)]</code>
```

Comment:
We should add another comment here explaining why duplicates aren't allowed with the realloc api.

Reply:
We should also add documentation that explains why its unsafe to manually realloc, since the solana_program sdk doesn't have any checks that address this case.